File: crud-framework-core/src/main/java/com/antelopesystem/crudframework/components/startup/PostStartUp.java
package com.antelopesystem.crudframework.components.startup;

import java.lang.annotation.ElementType;
import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;
import java.lang.annotation.Target;

/**
 * Created by Shani on 27/12/2018.
 */
@Target(ElementType.METHOD)
@Retention(RetentionPolicy.RUNTIME)
public @interface PostStartUp {

	int priority() default 0;
}

File: crud-framework-core/src/main/java/com/antelopesystem/crudframework/components/startup/PostStartUp.kt
package com.antelopesystem.crudframework.components.startup

import java.lang.annotation.Retention
import java.lang.annotation.RetentionPolicy

/**
 * Created by Shani on 27/12/2018.
 */
@Target(AnnotationTarget.FUNCTION, AnnotationTarget.PROPERTY_GETTER, AnnotationTarget.PROPERTY_SETTER)
annotation class PostStartUp(val priority: Int = 0)